frontend/src/config/app-config.ts

```diff
@@ -46,6 +46,13 @@ interface Config {
      * @description This is used to customize the appearance of your application.
      */
     THEME_CONFIGURATION: string;
+
+    /**
+     * The PostHog key and host for tracking events.
+     * @description This is used to send events to PostHog for analytics.
+     */
+    POSTHOG_KEY: string;
```

Comment:
If you want to see Posthog in action on the actual demo environments (capture some real data)
This setting needs to be adjustable in the helm charts. Have a look at `helm/templates/configmap_frontend.yaml` and `helm/values.yaml`